lib/Installation/NTPConfiguration/NTPConfigurationPage.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to act on the
+#          ntp Configuration page.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Installation::NTPConfiguration::NTPConfigurationPage;
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init($args);
+}
+
+sub init {
+    my $self = shift;
+    $self->{txb_ntp_server} = $self->{app}->textbox({id => '"Y2Caasp::Widgets::NtpServer"'});
+    $self->{btn_next} = $self->{app}->button({id => 'next'});
+    return $self;
+}
+
+sub is_shown {
+    my ($self) = @_;
+    return $self->{txb_ntp_server}->exist({timeout => 200});
+}
+
+sub get_ntp_value {
+    my ($self) = @_;
+    $self->get_ntp_configuration_page();
+    return $self->{txb_ntp_server}->value();
+}
+
+sub get_ntp_configuration_page {
+    my ($self) = @_;
+    die "Ntp configuration page is not displayed" unless $self->is_shown();
+}
+
+sub accept_ntp_config {
```

Comment:
"next" would suffice, it is redundant to use the whole name, and "accept" would be ok, but as this never changes, "next()" is very clear. But it is just a recommendation.